src/utils.rs

```diff
@@ -199,32 +199,58 @@ pub fn projv(a: &Vector3<f64>, b: &Vector3<f64>) -> Vector3<f64> {
     b * a.dot(b) / b.dot(b)
 }
 
-/// Computes the RSS state errors in two provided vectors
+/// Computes the Root Sum Squared (RSS) state errors between two provided vectors.
+///
+/// # Arguments
+///
+/// * `prop_err` - A vector representing the propagated error.
+/// * `cur_state` - A vector representing the current state.
+///
+/// # Returns
+///
+/// * A f64 value representing the RSS state error.
```

Comment:
```suggestion
/// A f64 value representing the RSS state error.
```
(minor) I don't think we need this to be a bullet point

Reply:
Done.